docs/docs/.vuepress/config.js

```diff
@@ -33,156 +52,94 @@ module.exports = {
     editLinkText: "Help us improve this page! Submit an edit on Github",
     sidebar: [
       "/",
-      {
-        title: "Quickstart",
-        collapsable: false,
-        children: [
-          "/quickstart/",
-          "/quickstart/hello-world/",
-          "/quickstart/hello-name/",
-          "/quickstart/make-http-request/",
-          "/quickstart/using-npm-packages/",
-          "/quickstart/add-data-to-google-sheets/",
-          "/quickstart/end-workflow-early/",
-          "/quickstart/use-managed-auth-in-code/",
-          "/quickstart/run-workflow-on-a-schedule/",
-          "/quickstart/email-yourself/",
-          "/quickstart/real-world-example/",
-          "/quickstart/next-steps/",
-        ],
-      },
+      "/quickstart/",
       {
         title: "Workflows",
-        collapsable: false,
-        children: [
+        children: 
+        [
           "/workflows/",
           "/workflows/steps/",
           "/workflows/steps/triggers/",
-          "/components/actions/",
-          "/workflows/steps/code/",
-          "/workflows/steps/params/",
-          "/workflows/steps/code/state/",
-          "/workflows/steps/code/nodejs/http-requests/",
```

Comment:
Did you get a chance to edit these docs (HTTP requests, working with files) and add them to the new Node.js code section?

Reply:
Yes, all of these sections are added to Node JS docs:

* HTTP requests - `/docs-v2/code/nodejs/#making-http-requests-from-your-workflow`
* State -` /docs-v2/code/nodejs/#managing-state`
* Code - `/docs-v2/code/nodejs
* Params - `/code/nodejs/#passing-parameters-to-code-steps` (this particular section links to a more enriched section in `/workflows/steps/#passing-data-to-steps-step-parameters`